adam-core/src/main/scala/org/bdgenomics/adam/rich/RichGenotype.scala

```diff
@@ -38,4 +38,36 @@ class RichGenotype(val genotype: Genotype) {
       case _ => GenotypeType.NO_CALL
     }
   }
+
+
+  /**
+    * True if all observed alleles are the same (regardless of whether they are ref or alt); if any alleles are no-calls, this method will return false.
+    */
+  def isHom: Boolean = { isHomRef || isHomVar }
+
+  /**
+    * True if all observed alleles are ref; if any alleles are no-calls, this method will return false.
+    */
+  def isHomRef: Boolean = { getType == GenotypeType.HOM_REF }
```

Comment:
Professing some Scala ignorance here, is ```==``` ok for enum values across different JVMs and serialization?